Nos/Service/PushNotificationService.swift

```diff
@@ -98,8 +98,7 @@ import Combine
         let userMentionsFilter = Filter(
             kinds: [.text, .longFormContent, .like], 
             pTags: [authorKey], 
-            limit: 50,
-            keepSubscriptionOpen: true
+            limit: 50
```

Comment:
I'm not understanding why we don't want to keep this subscription open. It seems like we _always_ want to know immediately when a user is mentioned. And then I don't get the limit of 50, but certainly I'm missing something.

Reply:
I think you are right. The `RelayView` keeps a similar subscription open, but maybe it's better to keep this one open. I switched them and I removed .longFormContent and .like from this one because we don't show notifications for those yet.